packages/grafana-data/src/utils/logs.test.ts

```diff
@@ -1,4 +1,5 @@
-import { LogLevel } from '../types/logs';
```

Comment:
As we have moved `LogsSortOrder`and `sorting functions` from core to grafana/data, we are moving the associated tests as well.